test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/FreezeUnfreezeTokenPrecompileSuite.java

```diff
@@ -483,4 +486,44 @@ private HapiSpec isFrozenHappyPathWithLocalCall() {
                                 }))
                 .then();
     }
+
+    private HapiSpec isFrozenHappyPathWithAliasLocalCall() {
+        final AtomicReference<TokenID> vanillaTokenID = new AtomicReference<>();
+        final AtomicReference<String> spenderAutoCreatedAccountId = new AtomicReference<>();
+        final String spenderAlias = "spenderAlias";
```

Comment:
I believe "spender" is not so applicable name in this context. It could be just "account", so we can rename the variables to "autoCreatedAccountId", "accountAlias", etc.